tests/sys/gir-secret.toml

```diff
@@ -11,6 +11,9 @@ external_libraries = [
 ]
 
 ignore = [
+   "Secret.MAJOR_VERSION",
+   "Secret.MINOR_VERSION",
+   "Secret.MICRO_VERSION",
```

Comment:
Based on [semver spec](https://semver.org/), it should be called `PATCH`, but I prefer this wording. :3